src/main/java/uk/gov/companieshouse/confirmationstatementapi/model/dao/ConfirmationStatementSubmissionDataDao.java

```diff
@@ -0,0 +1,16 @@
+package uk.gov.companieshouse.confirmationstatementapi.model.dao;
+
+import org.springframework.data.mongodb.core.mapping.Field;
+
+public class ConfirmationStatementSubmissionDataDao {
+    @Field("statement_of_capital_data")
```

Comment:
appreciate you may be compressing the dao and json into one singular class, but this field should be called data, not statementOfCapitalData